src/settings/SupportConfig.cpp

```diff
@@ -0,0 +1,75 @@
+//Copyright (c) 2020 Ultimaker B.V.
+//CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#include "SupportConfig.h"
+
+#include "types/Ratio.h"
+
+namespace cura
+{
+
+bool SupportConfig::LowerLayer(const size_t& layer_no)
```

Comment:
Code style: Methods use lower camel case.
```suggestion
bool SupportConfig::lowerLayer(const size_t& layer_no)
```
This also goes for the rest of the methods in this class.